src/wp-admin/includes/class-walker-nav-menu-edit.php

```diff
@@ -100,52 +101,59 @@ public function start_el( &$output, $item, $depth = 0, $args = array(), $id = 0
 		} elseif ( isset( $item->post_status ) && 'draft' == $item->post_status ) {
 			$classes[] = 'pending';
 			/* translators: %s: title of menu item in draft status */
-			$title = sprintf( __('%s (Pending)'), $item->title );
+			$title = sprintf( __( '%s (Pending)' ), $item->title );
 		}
 
 		$title = ( ! isset( $item->label ) || '' == $item->label ) ? $title : $item->label;
 
 		$submenu_text = '';
-		if ( 0 == $depth )
+		if ( 0 == $depth ) {
 			$submenu_text = 'style="display: none;"';
+		}
 
 		?>
-		<li id="menu-item-<?php echo $item_id; ?>" class="<?php echo implode(' ', $classes ); ?>">
+		<li id="menu-item-<?php echo $item_id; ?>" class="<?php echo implode( ' ', $classes ); ?>">
 			<div class="menu-item-bar">
 				<div class="menu-item-handle">
 					<span class="item-title"><span class="menu-item-title"><?php echo esc_html( $title ); ?></span> <span class="is-submenu" <?php echo $submenu_text; ?>><?php _e( 'sub item' ); ?></span></span>
 					<span class="item-controls">
 						<span class="item-type"><?php echo esc_html( $item->type_label ); ?></span>
 						<span class="item-order hide-if-js">
-							<a href="<?php
+							<a href="
```

Comment:
This puts a line break inside the attribute....not sure it matters.

Reply:
We could consider adding this backport:
https://core.trac.wordpress.org/changeset/45614

Reply:
Yes

Reply:
I'll get that added then - it does look better.

Reply:
PR updated